include/SFML/Audio/ChorusEffect.hpp

```diff
@@ -0,0 +1,232 @@
+    /// \brief Copy constructor
+    ///
+    /// \param copy Instance to copy
+    ///
+    ////////////////////////////////////////////////////////////
+    ChorusEffect(const ChorusEffect& copy);
+
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Waveform enum
+    ///
+    /// \see setWaveform()
+    ////////////////////////////////////////////////////////////
+    enum Waveform
+    {
+        Sine,
+        Triangle
+    };
+
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the current waveform value
+    ///
+    /// This property sets the waveform shape of the LFO that controls
+    /// the delay time of the delayed signals.
+    /// 
+    /// \param waveform new Waveform to set. Defaults to Waveform::Triangle
+    ////////////////////////////////////////////////////////////
+    void setWaveform(Waveform waveform);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the current phase value
+    ///
+    /// This property controls the phase difference between the left and
+    /// right LFOs. At zero degrees the two LFOs are synchronized. Use
+    /// this parameter to create the illusion of an expanded stereo field
+    /// of the output signal.
+    /// 
+    /// \param angle The angle of phase to set, between -180 and 180. Defaults to 90
+    ////////////////////////////////////////////////////////////
+    void setPhase(sf::Int32 angle);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the current rate value
+    ///
+    /// This property sets the modulation rate of the LFO that controls
+    /// the delay time of the delayed signals.
+    /// 
+    /// \param rate new rate to set in hz, between 0.f and 10.f. Defaults to 1.1f
+    ////////////////////////////////////////////////////////////
+    void setRate(float rate);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the current depth value
+    ///
+    /// This property controls the amount by which the delay time is modulated by the LFO
+    /// 
+    /// \param depth new depth to set, between 0.f and 1.f. Defaults to 0.1f
+    ////////////////////////////////////////////////////////////
+    void setDepth(float depth);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the current feedback value
+    ///
+    /// This property controls the amount of processed signal that is fed back to the
+    /// input of the chorus effect. Negative values will reverse the phase of the feedback
+    /// signal. At full magnitude the identical sample will repeat endlessly. At lower
+    /// magnitudes the sample will repeat and fade out over time. Use this parameter to
+    /// create a "cascading" chorus effect.
+    /// 
+    /// \param amount new feedback amount to set, between -1.f and 1.f. Defaults to 0.25f
```

Comment:
This review contains a lot of magic variables like "0.25f" or "0.404f". Can you please explain where they come from? I didn't find why they're significant.
(It's especially not clear why parameter ranges are the way they are: [some_magic_number_1;some_magic_number_2])

Reply:
These are the default values as laid out in the OpenAL specification. I guess the comments should be at least updated to state that fact.

Reply:
Yep, reference to OpenAL spec would be enough